titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/workflow/DefaultRelocationWorkflowExecutor.java

```diff
@@ -183,7 +183,7 @@ public void shutdown() {
 
     private void nextRelocationStep(ExecutionContext executionContext) {
         long count = executionContext.getExecutionId().getTotal();
-        boolean descheduling = lastDeschedulingTimestamp + configuration.getDeschedulingIntervalMs() > titusRuntime.getClock().wallTime();
+        boolean descheduling = lastDeschedulingTimestamp + configuration.getDeschedulingIntervalMs() <= titusRuntime.getClock().wallTime();
```

Comment:
make a utility function? I have some private ones in some places but maybe a general purpose one would reduce this type of mistake.

Reply:
Good idea. I have added a `isPast` method to the `Clock` interface.